windows/src/ext/cef4delphi/packages/res/00-CreateLazarusResources.bat

```diff
@@ -0,0 +1,7 @@
+e:\lazarus\tools\lazres.exe ..\..\source\res\tbufferpanel.lrs tbufferpanel.png
+e:\lazarus\tools\lazres.exe ..\..\source\res\tcefservercomponent.lrs tcefservercomponent.png
+e:\lazarus\tools\lazres.exe ..\..\source\res\tcefwindowparent.lrs tcefwindowparent.png
+e:\lazarus\tools\lazres.exe ..\..\source\res\tcefworkscheduler.lrs tcefworkscheduler.png
+e:\lazarus\tools\lazres.exe ..\..\source\res\tchromium.lrs tchromium.png
+e:\lazarus\tools\lazres.exe ..\..\source\res\tchromiumwindow.lrs tchromiumwindow.png
```

Comment:
Two questions here;
1. Is downloading/configuring lazarus a new step for Windows developers?
2. Should the lazarus path be configurable (besides `e:\lazarus`)?